### DISCUSSION TITLE
Can't install EA app in Lutris

### DISCUSSION THREAD

**Discussion by Jormagg2** [lutris/lutris#5561] [Q&A]
TypeError("wineexec() got an unexpected keyword argument 'return_code'")

I'm having a hard time trying to fix this :(((((( can someone help me ?

**Comment by danieljohnson2** [comment]
This should have been a proper issue report, not  discussion. With a log, I could tell what version of Lutris you have!

This sounds like #4177, which was fixed years ago. You may be trying to use an old version of Lutris, when the install script is for a more recent one. The current version is 0.5.17, and there are instructions for getting it [here](https://lutris.net/downloads).

Make sure you are on the latest Lutris.

**Comment by Jormagg2** [comment]
Sorry it's my first time around here, i' ll try fixing it with the info u
gave me,  thanks a lot !
